.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "useTabs": true,
```

Comment:
useTabs:false!!

Reply:
Sorry, forgot to squash one of the commits I dropped